amundsen_application/static/js/components/DashboardPage/QueryListItem/index.tsx

```diff
@@ -1,27 +1,34 @@
 import * as React from 'react';
 import { OverlayTrigger, Popover } from 'react-bootstrap';
 
+import { ResourceType } from 'interfaces';
+
+import { getSourceDisplayName, getSourceIconClass } from 'config/config-utils';
+
 import './styles.scss';
 
 export interface QueryListItemProps {
+  name: string;
+  product: string;
   text: string;
   url: string;
-  name: string;
 }
 
 type GoToDashboardLinkProps = {
+  product: string;
   url: string;
 };
 
 const QUERY_LABEL = 'Query';
-const MODE_LINK_TOOLTIP_TEXT = 'View in Mode';
+const LINK_TOOLTIP_TEXT = 'View in';
 const LOADING_QUERY_MESSAGE = 'Loading Query Component, please wait...';
 
 const LazyComponent = React.lazy(() => import('./CodeBlock'));
 
-const GoToDashboardLink = ({ url }: GoToDashboardLinkProps) => {
+const GoToDashboardLink = ({ product, url }: GoToDashboardLinkProps) => {
+  const productName = getSourceDisplayName(product, ResourceType.dashboard);
   const popoverHoverFocus = (
-    <Popover id="popover-trigger-hover-focus">{MODE_LINK_TOOLTIP_TEXT}</Popover>
+    <Popover id="popover-trigger-hover-focus">{`${LINK_TOOLTIP_TEXT} ${productName}`}</Popover>
```

Comment:
does `{{LINK_TOOLTIP_TEXT} {productName}}` work?

Reply:
No it would not work, that would be invalid syntax. This is a template string in javascript, and it is sort of the equivalent of `f-strings` in python. More info: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
thanks a lot